cfgov/regulations3k/models/django.py

```diff
@@ -87,7 +87,7 @@ class EffectiveVersion(models.Model):
     authority = models.CharField(max_length=255, blank=True)
     source = models.CharField(max_length=255, blank=True)
     effective_date = models.DateField(blank=True, null=True)
-    acquired = models.DateField(blank=True, null=True)
+    created = models.DateField(blank=True, null=True)
```

Comment:
I realize this would complicate the migration substantially, but I wonder if this should have a default of `date.today()`? That way creating a new, blank effective version always has the creation date.

Reply:
I'll add this if we need it. The importer and copy buttons were already resulting in created today(), but a default makes sense.